### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by Yu-xm** [DLCV-BUAA/TinyLLaVABench#53]
Can you share the loss curve during the training process?

**Comment by Yu-xm** [comment]
Including pretrain and finetune

**Comment by cooleel** [comment]
They shared the training loss curves for LLaVA-v1.5-13B at https://github.com/haotian-liu/LLaVA/issues/579#issuecomment-1763707042
Mine look the same.
